pkg/systemd/terminal.jsx

```diff
@@ -71,19 +79,28 @@ const _ = cockpit.gettext;
             if (this.state.channel)
                 terminal = (<Terminal ref="terminal"
                      channel={this.state.channel}
+                     theme={this.state.theme}
                      onTitleChanged={this.onTitleChanged} />);
             else
                 terminal = <span>Loading...</span>;
 
             return (
                 <div className="console-ct-container">
-                    <div className="panel-heading">
+                    <div className="panel-heading terminal-group">
                         <tt className="terminal-title">{this.state.title}</tt>
-                        <button ref="resetButton"
-                             className="btn btn-default pull-right"
-                             onClick={this.onResetClick}>{_("Reset")}</button>
+                        <div>
```

Comment:
Why does this need an extra div wrapper?

Reply:
Because of flex. I want terminal title to be on left and these two buttons to be on right and flex can nicely spread them. However I want to keep buttons together. Probably this can be done in more elegant way, but someone with more skills with css would need to give me some hint :)

Reply:
Ack, thanks! Fine then.